PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -1059,6 +1059,13 @@ void AliConvEventCuts::LoadGammaPtReweightingHistosMCFromFile() {
 ///________________________________________________________________________
 int AliConvEventCuts::InitializeMapPtWeightsAccessObjects()
 {
+  auto multiplyTF1ByX = [](TF1 const &theF){
+    TF1 *lResult = new TF1(Form("%s_multByX", theF.GetName()),
+                            Form("x*(%s)", theF.GetTitle()),
```

Comment:
Shouldt it be GetExpFormula here?

Reply:
why? I dont see any "exp" in these lines

Reply:
Thanks for the comment @Joshua!
I changed to GetExpFormula because it reads nicer like this. For TF1 defined through expressions there is no difference except whitespace removed:

f->GetTitle()
(const char *) "[0]*TMath::Exp(-(TMath::Sqrt(x*x+0.1349770000*0.1349770000)-0.1349770000)/[1]) + [2]/(TMath::Power(1+x*x/([3]*[3]*[4]),[4]) ) + [5]/(TMath::Power(1+x*x/([6]*[6]*[7]),[7]) )"

f->GetExpFormula()
(TString) "[p0]*TMath::Exp(-(TMath::Sqrt(x*x+0.1349770000*0.1349770000)-0.1349770000)/[p1])+[p2]/(TMath::Power(1+x*x/([p3]*[p3]*[p4]),[p4]))+[p5]/(TMath::Power(1+x*x/([p6]*[p6]*[p7]),[p7]))"






f->GetExpFormula()
(TString) "[p0]*TMath::Exp(-(TMath::Sqrt(x*x+0.1349770000*0.1349770000)-0.1349770000)/[p1])+[p2]/(TMath::Power(1+x*x/([p3]*[p3]*[p4]),[p4]))+[p5]/(TMath::Power(1+x*x/([p6]*[p6]*[p7]),[p7]))"

Reply:
Thanks!